scipy/linalg/flapack_sym_herm.pyf.src

```diff
@@ -302,6 +302,46 @@ end subroutine <prefix2c>heevd_lwork
    end subroutine <prefix>sytrf_lwork
 
 
+   subroutine <prefix>sytrs(n,nrhs,a,lda,ipiv,b,ldb,info,lower)
+   
+   ! Solve A * X = B for symmetric A matrix after calling ?sytrf
+   
+     callstatement (*f2py_func)((lower?"L":"U"),&n,&nrhs,a,&lda,ipiv,b,&ldb,&info)
```

Comment:
Does `(lower?"L":"U")` effectively change this to a boolean argument?

Reply:
Yes, this is one of those annoying parts of these wrappers that they are full of inconsistencies. Here I just followed the pair function `he/sytrf` style.